po/de.po

```diff
@@ -295,15 +295,19 @@ msgstr ""
 msgid "ARCHITECTURE"
 msgstr ""
```

Comment:
IMO many elements missing and some translations could be improved. ... help wanted?

Reply:
https://hosted.weblate.org/projects/linux-containers/lxd/